src/GraphQL/Utilities/SchemaBuilder.cs

```diff
@@ -12,7 +12,9 @@ namespace GraphQL.Utilities
     /// </summary>
     public class SchemaBuilder
     {
+#pragma warning disable CS1591 // Missing XML comment for publicly visible type or member
         protected readonly Dictionary<string, IGraphType> _types = new Dictionary<string, IGraphType>();
```

Comment:
This should be fixed!!!  Was it intended to be `internal protected` ?

Reply:
I don't know.